test/inductor/test_torchinductor.py

```diff
@@ -5543,11 +5543,46 @@ def fn(x, p1, p0):
             o = torch.where(x, p1, p0)
             return o
 
+        # https://github.com/pytorch/pytorch/issues/94725
+        class Repro(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.register_buffer(
+                    "_tensor_constant0", torch.randn([], dtype=torch.float32)
+                )
+
+            def forward(self, arg0_1, arg1_1):
+                convert_element_type = torch.ops.prims.convert_element_type.default(
+                    arg1_1, torch.bool
+                )
+                bitwise_not = torch.ops.aten.bitwise_not.default(convert_element_type)
+                _tensor_constant0 = self._tensor_constant0
+                lift_fresh_copy = torch.ops.aten.lift_fresh_copy.default(
+                    _tensor_constant0
+                )
+                where = torch.ops.aten.where.self(bitwise_not, lift_fresh_copy, arg0_1)
+                return (where, bitwise_not)
+
         self.common(
             fn,
             (torch.tensor([[True]]), torch.rand(13, 7, 3), torch.rand(1, 1)),
         )
 
+        if not torch._dynamo.config.dynamic_shapes:
+            args = [
+                ((1, 24, 512, 512), (6291456, 262144, 512, 1), torch.float32, "cpu"),
+                ((1, 1, 512, 512), (262144, 262144, 512, 1), torch.uint8, "cpu"),
+            ]
```

Comment:
Can we make the inputs smaller? We can also simply call `rand`, as these strides are contiguous.

Reply:
Yes, changed to smalled input.